src/applications/mhv/secure-messaging/components/MessageList/CernerFacilityAlert.jsx

```diff
@@ -26,7 +26,7 @@ const CernerFacilityAlert = () => {
     <>
       {cernerFacilities?.length > 0 && (
         <va-alert
-          className="vads-u-margin-bottom--2"
+          class="vads-u-margin-bottom--2"
```

Comment:
Should this be className instead?

Reply:
I find that `className` has no effect for some/most VA Design library components